client/src/test/java/com/netflix/conductor/client/task/WorkflowTaskCoordinatorTests.java

```diff
@@ -95,8 +97,8 @@ public void testTaskException() {
 				.withTaskClient(client)
 				.withWorkerNamePrefix("test-worker-")
 				.build();
-		when(client.batchPollTasksInDomain(anyString(), anyString(), anyString(), anyInt(), anyInt())).thenReturn(ImmutableList.of(new Task()));
-		when(client.ack(anyString(), anyString())).thenReturn(true);
+		when(client.batchPollTasksInDomain(anyString(), isNull(), anyString(), anyInt(), anyInt())).thenReturn(ImmutableList.of(new Task()));
```

Comment:
Why is the second argument `domain` changed to isNull?

Reply:
The behavior of `anyString()` changed so that it no longer matches `null`, and `null` is the actual value that was causing the stub to match in this case.
That conclusion was particularly difficult to arrive at, so while `any()` (which also matches `null`) would work, I instead went with `isNull()` to self-document and prevent the same confusion going forward.

Reply:
Sounds good, thanks for the explanation. Please rebase against dev and resolve conflicts and we are good to merge.

Reply:
Rebase has been pushed 👍